server/requirements.txt

```diff
@@ -1,12 +1,12 @@
 argparse==1.2.1
-Flask==0.10.1
-Flask-Login==0.2.11
-Flask-Mail==0.9.1
-Flask-Migrate==1.2.0
-Flask-SQLAlchemy==2.0
-Flask-Security==1.7.4
-Flask-Testing==0.4.2
-Flask-WTF==0.10.3
+Flask>=0.10.1
```

Comment:
we've had trouble in the past with incompatible libs due to flexible versions like this (mainly with Flask-Security).  i recommend reverting or pinning to the new exact versions (after testing that they work together).